.standard.yml

```diff
@@ -0,0 +1 @@
+parallel: true
```

Comment:
This is literally the only configuration we need for standard. And we don't even need it; it just speeds things up.

Reply:
This is *awesome*.